README.md

```diff
@@ -239,6 +240,29 @@ Input: I am foo test a test
 Output: I am hi a hi
 ```
 
+### Example for `regex_replacement_list`
+
+```
+regex_replacement_list = [
+  # Split glued sentences
+  ["\\ [a-z]{3,}\\.[A-Z][a-z]{2,}\\ ", ".", ". "],
+  
+  # Split long sentences
+  ["\\b(?:\\S+\\s+){15,}\\S+[.!?]", ", but ", ". But "],
+]
+```
+
+This will find words that glue two sentences and will add a space to un-glue them. 
+And will split a long sentence in two smaller.
```

Comment:
I think this is where it is most worthwhile: If the article does not have enough sentences to select from (<3) because of the rules, especially max_words and/or max_characters. At that time, this algorithm can kick in and try to produce split sentences.

There is no way for us to know if pre-split or post-split can produce more "valuable" sentences. But many of the "sub-sentences" might be simple introductory wording etc.

Reply:
I agree, and that's exactly why I would prefer a short sentence explaining that, so people don't just blindly copy. If we have an indication that it works in all corpuses, then we could also just do it by default.

Reply:
Added a note on sentence splitting.